src/ontology/mondo-edit.obo

```diff
@@ -293933,20 +293913,36 @@ property_value: confidence "0.7021739130434781" xsd:double
+synonym: "Au-Kline syndrome" EXACT [OMIM:616580, Orphanet:453504]
+synonym: "AUKS" EXACT ABBREVIATION [MONDO:Lexical, OMIM:616580]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and intellectual disability" RELATED [GARD:0004064]
+synonym: "congenital hydronephrosis with cleft palate, characteristic facies, hypotonia, and mental retardation" RELATED DEPRECATED [GARD:0004064]
+synonym: "hydronephrosis, congenital, with cleft palate, characteristic facies, hypotonia, and intellectual disability" EXACT [OMIM:604916]
+synonym: "hydronephrosis, congenital, with cleft palate, characteristic facies, hypotonia, and mental retardation" EXACT DEPRECATED [OMIM:604916]
+synonym: "neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome due to a point mutation" EXACT CLINGEN_PREFERRED [Orphanet:453504]
+synonym: "Okamoto syndrome" EXACT [OMIM:604916]
+xref: ICD10CM:Q87.8 {source="Orphanet:2729/attributed", source="Orphanet:2729/ntbt", source="Orphanet:2729"}
+xref: MESH:C565736 {source="MONDO:equivalentTo"}
+xref: OMIM:604916 {source="MONDO:equivalentObsolete", source="Orphanet:2729/e", source="Orphanet:2729"}
 xref: OMIM:616580 {source="MONDO:equivalentTo", source="Orphanet:453504", source="Orphanet:453504/e"}
+xref: Orphanet:2729 {source="OMIM:604916"}
 xref: Orphanet:453504 {source="MONDO:equivalentTo"}
+xref: SCTID:722065002 {source="MONDO:equivalentTo"}
+xref: UMLS:C1858043 {source="MONDO:ncbi_mim2gene_medline", source="OMIM:604916"}
 xref: UMLS:C4225274 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline"}
-xref: UMLS:CN237748 {source="MONDO:equivalentTo"}
-is_a: MONDO:0018681 {source="Orphanet:453504"} ! neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome
+xref: UMLS:CN237748 {source="MONDO:equivalentObsolete"}
+is_a: MONDO:0015159 {source="Orphanet:2729"} ! multiple congenital anomalies/dysmorphic syndrome-intellectual disability
+is_a: MONDO:0018681 {source="Orphanet:453504", source="https://orcid.org/0000-0001-5208-3432"} ! neurodevelopmental disorder-craniofacial dysmorphism-cardiac defect-hip dysplasia syndrome
+relationship: excluded_subClassOf MONDO:0000508 {source="MONDO:Redundant", source="Orphanet:2729"} ! syndromic intellectual disability
+relationship: has_characteristic MONDO:0021136 {source="MONDO:0015162", source="MONDO:0015983"} ! rare
 relationship: has_characteristic SO:1000008 ! point_mutation
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/5044 {source="MONDO:mim2gene_medgen"} ! HNRNPK
-property_value: confidence "6.5" xsd:double
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/5588" xsd:anyURI
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6335" xsd:anyURI
+property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/4064/okamoto-syndrome" xsd:anyURI {source="GARD:0004064"}
```

Comment:
Why was this transferred as a `seeAlso` vs. a `subset` as found in the merge obsolete term 
(`subset: gard_rare {source="GARD:0004064"}`)? The subset `ordo_malformation_syndrome` was transferred as a `subset`.

Reply:
we don't have this documented in the documentation, maybe we should add this somewhere.

we used to add seeAlso for any URL (including the github URLs) but recently we started using the term tracker item annotation instead of seeAlso, but we still use seeAlso for other URLs, like GARD.

Reply:
yeah, the difference in how the two are handled is what caught my eye and is why I asked. 

NOTE: there is also `http://www.w3.org/2000/01/rdf-schema#seeAlso` that a number of ontologies in OLS use as well as `http://www.geneontology.org/formats/oboInOwl#seeAlso` that a smaller number of ontologies use that could be good to use instead of free text `seeAlso` and hopefully would not break anything downstream.